hack/local-up-cluster.sh

```diff
@@ -599,14 +599,24 @@ elif [[ "${START_MODE}" == "nokubelet" ]]; then
 fi
 
 if [[ "${START_MODE}" != "kubeletonly" ]]; then
+  auth_arg=""
+  if [[ -n "${ALLOW_ANY_TOKEN:-}"  ]]; then
+      if [[ -n "${KUBECONFIG_TOKEN:-}"  ]]; then
+          auth_arg=${KUBECONFIG_TOKEN}
+      else
+          auth_arg="--token=admin/system:masters"
```

Comment:
How about `system:admin/system:masters`?  When we start creating client certificates for core system roles, I think we'll continue the pattern of `system` prefixes denoting managed/curated things.